src/components/ReportActionItem/IOUAction.js

```diff
@@ -30,9 +34,21 @@ const propTypes = {
         hasOutstandingIOU: PropTypes.bool.isRequired,
     }),
 
+    /** IOU report data object */
+    iouReport: PropTypes.shape({
+        /** The currency of the iouReport */
+        currency: PropTypes.number,
+    }).isRequired,
+
+    /** Array of report actions for this report */
+    reportActions: PropTypes.objectOf(PropTypes.shape(reportActionPropTypes)).isRequired,
+
     /** Whether the IOU is hovered so we can modify its style */
     isHovered: PropTypes.bool,
 
+    network: {
```

Comment:
qq: again we have `networkPropTypes` defined, so any specific reason we're defining again instead of importing?